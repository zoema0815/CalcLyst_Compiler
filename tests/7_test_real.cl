int main () {
    Real r1;
    Real r2;
    r1 = 3.5r;
    r2 = 3.8r;
    printReal(r1);
    printReal(r2);
    printReal(r1 + r2);
    printReal(r1 - r2);
    printReal(r1 * r2);
    printReal(r1 / r2);
    return 0;
}