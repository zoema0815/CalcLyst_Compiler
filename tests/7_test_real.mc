int main () {
    Real r;
    r = 3.5r;
    printReal(r);
    return 0;
}