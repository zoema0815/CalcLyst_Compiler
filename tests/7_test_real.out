3.5r
3.8r
7.300000r
-0.300000r
13.300000r
0.921053r
